src/asus-stylus.py

```diff
@@ -18,11 +20,35 @@
 log = logging.getLogger('Pen')
 log.setLevel(os.environ.get('LOG', 'INFO'))
 
-layout_name = 'SA201H'
-if len(sys.argv) > 1:
-    layout_name = sys.argv[1]
+parser = argparse.ArgumentParser('Asus Stylus Driver', description='Asus stylus supplement driver')
+parser.add_argument('-c', '--config', help='override config file path', type=str, metavar='path')
+parser.add_argument('-l', '--layout', help='override layout name', choices=['SA201H'], metavar='name')
```

Comment:
@SelfRef Does not `choices=['SA201H']` limit layout only to `SA201H` so it cancels the possibility of using a custom layout as argument?

Reply:
It does. It will prevent you from choosing a value that's not in this list. I think it could be set dynamically based on available layouts.

Do you expect the layout name to be any string? In that case I can just replace it with `type=str` and it will accept anything. 

A good idea would be to add another command line option to list available layouts.

Reply:
@SelfRef Please change that to `type=str`.

Reply:
@SelfRef

> A good idea would be to add another command line option to list available layouts.

Agree. Can you add it, please?